DeepLinkSDK/RouteHandler/DPLRouteHandler.h

```diff
@@ -51,6 +58,10 @@
  */
 - (UIViewController *)viewControllerForPresentingDeepLink:(DPLDeepLink *)deepLink;
 
+/**
+ returns the first `UINavigationController'
+ */
+- (UINavigationController *)navigationControllerForPresentingDeepLink:(DPLDeepLink *)deepLink;
```

Comment:
@wessmith That's in fact how I was doing it, but it seems to me it adds more complexity (responsibilities) to a subclass. Moreover, when you have a lot of `DPLRounteHandler` subclasses, it can be quickly become non-DRY. 

You can also argue that one can create a subclass-base-class, and all the `DPLRounteHandler` subclasses can inherit from it, but to me this isn't good neither.  

Please let me know what you think, thanks!!!